python/src/scipp/plot/plot.py

```diff
@@ -61,45 +63,43 @@ def _variable_to_data_array(variable):
     return sc.DataArray(data=variable, coords=coords)
 
 
-def _raise_plot_input_error(intype):
+def _ndarray_to_variable(ndarray):
     """
-    Raise an error on bad input type.
+    Convert a numpy ndarray to a Variable.
+    Fake dimension labels begin at 'x' and cycle through the alphabet.
     """
-    raise RuntimeError("plot: Unknown input type: {}. Allowed inputs are "
-                       "a Dataset, a DataArray, a Variable (and their "
-                       "respective proxies), and a dict of "
-                       "Variables or DataArrays.".format(intype))
+    dims = [f"axis-{i}" for i in reversed(range(len(ndarray.shape)))]
```

Comment:
Why `reversed`? I thought numpy's `axis 0 is the *outer* axis, i.e., showing up *first* in `shape`?

Reply:
Also `ndarray.ndim` instead of `len(shape)`?

Reply:
I felt it was counter-intuitive to have `axis-1` along the x axis and `axis-0` along the y axis (and i've gotten that comment from some of the imaging people also, that having `dims=['y', 'x']` was conuter-intuitive), but i think i've just been working with fortran for too long

Reply:
My comment was not about what should be on the axes, rather that `axis-0` should be what, e.g., `np.sum(a, axis=0)` sums. If we change that it would be quite confusing?

Printing a 2d numpy array also shows axis-1 (inner) horizontally, so this would be consistent.

Reply:
makes sense about the `np.sum(a, axis=0)`